cypress/integration/manual-aggregated-sync.spec.js

```diff
@@ -1,76 +1,136 @@
-import { dataTest } from "../support/utils";
+import ManualAggregateSyncPageObject from "../pageobjects/ManualAggregateSyncPageObject";
 
+/**
+ * Database: d2-docker-eyeseetea-2-30-datasync-sender
+ */
 context("Manual aggregated sync", function() {
+    const page = new ManualAggregateSyncPageObject(cy);
```

Comment:
LGTM, I would make the PageObject agnostic to "aggregated", "events" and "metadata" though and pass it here as an argument. I believe the same PageObject can be re-used for the three of them. (Similarly this would happen on SyncRules and History PageObjects)

Reply:
It is a possibility, however, I would not like to fall into premature abstractions since they can be misleading. If the page object can be used for all manual synchronizations I think it is something to review in the future PR when we create the next on-demand tests.

Reply:
Well, they're the same component with different models so the abstraction is already there but in the application code. I believe these PageObjects should mimic the application design and naming so it's consistent and easy to follow, anyway it's okay to leave it to a future PR.

Reply:
I agree with both your points. On the next iteration, once we write tests for the events/metadata/..., we can see the proper abstraction more clearly.

Reply:
In the next iteration for the moment I'm currently opting for a base class but it is not finished